tests/unit/aiplatform/test_datasets.py

```diff
@@ -22,6 +22,7 @@
 from unittest import mock
 from importlib import reload
 from unittest.mock import patch
+import pandas as pd
```

Comment:
Preference to add this either in a separate import block or include with pytest import above as a third party import block